src/routeGeneration/templates/express.hbs

```diff
@@ -24,16 +24,20 @@ const models: TsoaRoute.Models = {
     {{#each models}}
     "{{@key}}": {
         {{#if enums}}
```

Comment:
The way you are setting them up would mean I should change the TAs to look similar. ("dataType": "refType")